packages/crayons-core/src/components/datepicker/datepicker.tsx

```diff
@@ -98,6 +98,7 @@ export class Datepicker {
   @State() toMonth: number;
   @State() firstFocusElement: HTMLElement = null;
   @State() lastFocusElement: HTMLElement = null;
+  @State() popoverContentElement: HTMLElement;
```

Comment:
should this be a state or can we just have it as a private variable ?

Reply:
This has to be a state, else boundary is null when select is first rendered.

This is the same as the case in the thread here:
https://stackoverflow.com/questions/59729258/stenciljs-jsx-use-ref-in-as-a-prop